### PR TITLE
Added machine-learning tasks, fields and layout section to Phishing v2

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
NEW BRANCH: https://github.com/demisto/content/pull/5484

## Description
Added tasks to predict phishing type using the phishing ML model.
Added 3 new fields and will be populated from the playbook.
Added a section to the phishing v2 layout

## Screenshots
image

image

## Required version of Demisto
4.5

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [x] [JSON file](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/IncidentFields/incidentfield-dbotprediction.json) - line 8
- [x] [JSON file](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/IncidentFields/incidentfield-dbotpredictionprobability.json) - line 8
- [x] [JSON file](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/IncidentFields/incidentfield-dbottextsuggestionhighlighted.json) - line 8
- [x] [YML file](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/Playbooks/playbook-Phishing_Investigation_-_Generic_v2.yml) - lines 1799-1801, 1866-1867, 2149-2150

- [x] [CHANGELOG](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/Layouts/layout-details-Phishing_CHANGELOG.md)
- [x] [CHANGELOG](https://github.com/demisto/content/blob/026c66a460a1f62a0c010f543c647049ea144948/Playbooks/playbook-Phishing_Investigation_-_Generic_v2_CHANGELOG.md)

### CODE REVIEW

**Comment by idovandijk** [comment]
Failing due to changes in secrets and hooks/SDK. Can't get green build until fixed.

**Comment by yaron-libman** [comment]
Completed review.

File: IncidentFields/incidentfield-dbotprediction.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+	"id": "incident_dbotprediction",
+	"version": -1,
+	"modified": "2019-12-08T12:25:50.370130149Z",
+	"name": "DBotPrediction",
+	"ownerOnly": false,
+	"placeholder": "",
+	"description": "The incident type that was predicted by the machine-learning model.",
```
Comment (erezh31) [inline IncidentFields/incidentfield-dbotprediction.json]: incident type -> incident verdict

File: IncidentFields/incidentfield-dbotpredictionprobability.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+	"id": "incident_dbotpredictionprobability",
+	"version": -1,
+	"modified": "2019-12-08T12:27:36.654533693Z",
+	"name": "DBotPredictionProbability",
+	"ownerOnly": false,
+	"placeholder": "",
+	"description": "The confidence level, presented as a value between 0 and 1, of the predicted incident type by the machine-learning model.",
```
Comment (erezh31) [inline IncidentFields/incidentfield-dbotpredictionprobability.json]: incident type -> incident verdict

File: IncidentFields/incidentfield-dbottextsuggestionhighlighted.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+	"id": "incident_dbottextsuggestionhighlighted",
+	"version": -1,
+	"modified": "2020-01-01T10:06:16.935978178Z",
+	"name": "DBotTextSuggestionHighlighted",
+	"ownerOnly": false,
+	"placeholder": "",
+	"description": "Indicates the text that contributed to the decision made by the machine learning model.",
```
Comment (erezh31) [inline IncidentFields/incidentfield-dbottextsuggestionhighlighted.json]: the words in the text

File: Playbooks/playbook-Phishing_Investigation_-_Generic_v2_CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## [Unreleased]
-
+Added tasks that predict the phishing sub-type when a phishing ML model exists.
```
Comment (erezh31) [inline Playbooks/playbook-Phishing_Investigation_-_Generic_v2_CHANGELOG.md]: phishing verdict

**Comment by idovandijk** [comment]
Branch is destroyed. Working here https://github.com/demisto/content/pull/5484

**Comment by erezh31** [comment]
@idovandijk you can delete this branch

**Pull request closed without merging**